tests/java-example.yml

```diff
@@ -0,0 +1,15 @@
+---
+runner: command
+dir: java-example/java
+args: [mvn, compile, exec:java, -Dexec.mainClass=com.start.Connection]
```

Comment:
That does not actually _runs_ the example, only compiles it

Reply:
I believe that Maven plugin `exec:java` will build and then run it in the [same VM](https://www.digitalocean.com/community/tutorials/exec-maven-plugin-run-java-programs-maven-build).

See [here](https://github.com/FerretDB/dance/actions/runs/5178231009/jobs/9329409310#step:7:2262) for output from `listDatabases()`.